autoload/codeium.vim

```diff
@@ -89,7 +89,7 @@ endfunction
 
 function! codeium#Accept() abort
   let current_completion = s:GetCurrentCompletionItem()
-  return s:CompletionInserter(current_completion, current_completion.completion.text) 
+  return s:CompletionInserter(current_completion, current_completion.completion.text)
```

Comment:
why did this change

Reply:
Apparently that line ended in a single space

Reply:
pre-commit caught this

Reply:
got it